sprout/appliances/tasks.py

```diff
@@ -907,6 +907,9 @@ def appliance_power_on(self, appliance_id):
             appliance.set_status("Powering on.")
             appliance.provider_api.start_vm(appliance.name)
             self.retry(args=(appliance_id, ), countdown=20, max_retries=40)
+
+        # fixes time synchronization
+        appliance.ipapp.fix_ntp_clock()
```

Comment:
hm, I thought the rest of if/elif statements should lead to this destination. fixed.